<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeshManager</class>
 <widget class="QWidget" name="MeshManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Meshes</string>
  </property>
  <layout class="QGridLayout" name="meshManagerLayout">
   <item row="0" column="0">
    <widget class="QListWidget" name="meshList">
     <property name="movement">
      <enum>QListView::Free</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="4">
    <widget class="QPropertyEditorWidget" name="meshProperties"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="meshButtons">
     <item>
      <widget class="QToolButton" name="createMeshButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>New</string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::MenuButtonPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteMeshButton">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QListWidget" name="modifierList">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="modifierButtons">
     <item>
      <widget class="QToolButton" name="createModifierButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>New</string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::MenuButtonPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteModifierButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionBox">
   <property name="text">
    <string>Box</string>
   </property>
  </action>
  <action name="actionCone">
   <property name="text">
    <string>Cone</string>
   </property>
  </action>
  <action name="actionCylinder">
   <property name="text">
    <string>Cylinder</string>
   </property>
  </action>
  <action name="actionTube">
   <property name="text">
    <string>Tube</string>
   </property>
  </action>
  <action name="actionCapsule">
   <property name="text">
    <string>Capsule</string>
   </property>
  </action>
  <action name="actionDisk">
   <property name="text">
    <string>Disk</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPropertyEditorWidget</class>
   <extends>QTreeView</extends>
   <header>QPropertyEditor/QPropertyEditorWidget.h</header>
   <slots>
    <slot>addObject(QObject*)</slot>
    <slot>setObject(QObject*)</slot>
    <slot>updateObject(QObject*)</slot>
    <slot>removeObject(QObject*)</slot>
    <slot>setObject(QListWidgetItem*)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
